plugin/src/main/java/org/wildfly/plugins/bootablejar/maven/goals/DevBootableJarMojo.java

```diff
@@ -64,7 +64,8 @@ protected void configureCli(List<String> commands) {
     }
 
     private void configureScanner(Path deployments, List<String> commands) {
+        String deploymentPath = deployments.toString().replaceAll("\\x5c", "/");
```

Comment:
I think we should just escape it with another `/` so something like `deployments.toString().replace("/", "//");`.

Reply:
I've updated the PR to replace with double backslash.

Reply:
@doychin, can't you use simpler syntax replace("\\", "\\\\");

Reply:
it fails

Reply:
Do you by chance have a reproducer for this? It seems to work for me when I try the jaxrs example.
```
[standalone@localhost:9990 /] /subsystem=deployment-scanner/scanner=wildfly-jar-for-dev-mode:read-resource
{
    "outcome" => "success",
    "result" => {
        "auto-deploy-exploded" => false,
        "auto-deploy-xml" => true,
        "auto-deploy-zipped" => true,
        "deployment-timeout" => 600,
        "path" => "C:\\Users\\jperkins\\projects\\wildfly-jar-maven-plugin\\examples\\jaxrs\\target\\deployments",
        "relative-to" => undefined,
        "runtime-failure-causes-rollback" => false,
        "scan-enabled" => true,
        "scan-interval" => 1000
    }
}
```

Reply:
replaceAll uses regular expression.

Reply:
I think I see what happens here. When you write GitHub message and you use \\ (I wrote double backslash) you get only one in the message. 
;-)

Reply:
I've committed the change